scss/components/_ckeditor.scss

```diff
@@ -0,0 +1,9 @@
+.ckeditor-toolbar-configuration {
+  .ckeditor-button {
+    a {
```

Comment:
Because this is a `drupal-overrides` rule set, using an `a` selector is acceptable - but note that generally, a class selector is preferable to a tag selector. If you are building a component, give every element its own class following the `BEM` syntax.